po/fr.po

```diff
@@ -0,0 +1,2453 @@
+
+#: setzer/workspace/shortcutsbar/shortcutsbar_viewgtk.py:440
+msgid "List with Bold Labels"
+msgstr "Descriptions"
+
+#: setzer/workspace/shortcutsbar/shortcutsbar_viewgtk.py:453
+msgid "Objects"
+msgstr "Objets"
+
+#: setzer/workspace/sidebar/sidebar_viewgtk.py:106
+msgid "Package"
+msgstr "Paquets"
+
+#: setzer/workspace/sidebar/sidebar.py:41
+msgid "Greek Letters"
+msgstr "Lettres grecs"
+
+#: setzer/workspace/sidebar/sidebar.py:43
+msgid "Arrows"
+msgstr "Flèches"
+
+#: setzer/workspace/sidebar/sidebar.py:45
+msgid "Relations"
+msgstr "Relations"
+
+#: setzer/workspace/sidebar/sidebar.py:47
+msgid "Operators"
+msgstr "Opérations"
+
+#: setzer/workspace/sidebar/sidebar.py:49
+msgid "Misc. Math"
+msgstr "Divers symboles mathématiques"
+
+#: setzer/workspace/sidebar/sidebar.py:51
+msgid "Misc. Symbols"
+msgstr "Divers symboles"
+
+#: setzer/workspace/workspace_viewgtk.py:252
+msgid "No open documents"
+msgstr "Pas de documents ouverts"
+
+#: setzer/workspace/workspace.py:94
+#, python-brace-format
+msgid "Untitled Document {number}"
+msgstr "Document vierge {number}"
+
+#: data/org.cvfosammmm.Setzer.appdata.xml.in:7
+msgid "Setzer"
+msgstr "Setzer"
+
+#: data/org.cvfosammmm.Setzer.appdata.xml.in:8
+msgid "Simple yet full-featured LaTeX editor"
+msgstr "Éditeur LaTeX simple mais complet"
+
+#: data/org.cvfosammmm.Setzer.appdata.xml.in:9
+msgid "Cvfosammmm"
+msgstr "Cvfosammmm"
+
+#: data/org.cvfosammmm.Setzer.appdata.xml.in:12
+msgid ""
+"Setzer lets you Write LaTeX documents with an easy to use yet full-featured "
+"editor."
+msgstr ""
+"Setzer vous laisse écrire des documents LaTeX avec un éditeur simple mais "
+"complet."
+
+#: data/org.cvfosammmm.Setzer.appdata.xml.in:13
+msgid "Features include:"
+msgstr "Parmi les fonctionnalités :"
+
+#: data/org.cvfosammmm.Setzer.appdata.xml.in:15
+#: data/org.cvfosammmm.Setzer.appdata.xml.in:30
+msgid "Buttons and shortcuts for many LaTeX elements and special characters"
+msgstr ""
+"Boutons et raccourcis pour plusieurs éléments LaTeX et caractères spéciaux"
+
+#: data/org.cvfosammmm.Setzer.appdata.xml.in:16
+msgid "Comfortable document creation wizard"
+msgstr "Un assistant confortable pour la création de document"
+
+#: data/org.cvfosammmm.Setzer.appdata.xml.in:17
+#: data/org.cvfosammmm.Setzer.appdata.xml.in:38
+msgid "Dark mode"
+msgstr "Mode sombre"
+
+#: data/org.cvfosammmm.Setzer.appdata.xml.in:18
+#: data/org.cvfosammmm.Setzer.appdata.xml.in:42
+msgid "Helpful error messages in the build log"
+msgstr "Messages d'erreurs utiles dans le journal de construction"
+
+#: data/org.cvfosammmm.Setzer.appdata.xml.in:19
+msgid "Looks great on the Gnome desktop"
+msgstr "S'intègre bien avec le bureau Gnome"
+
+#: data/org.cvfosammmm.Setzer.appdata.xml.in:20
+msgid "Good screen to content ratio"
+msgstr "Bonne ergonomie"
+
+#: data/org.cvfosammmm.Setzer.appdata.xml.in:21
+msgid "Arguably the best .pdf viewer of any LaTeX editor"
+msgstr "Sûrement le meilleur visionneur PDF de n'importe quel éditeur LaTeX"
```

Comment:
I think "sûrement" is a bad translation of "arguably".
```suggestion
msgstr "Peut-être la meilleure visionneuse PDF parmi les éditeurs LaTeX"
```

Reply:
For this, now that I'm thinking of it "Sans doute" might be even better.
Like in: "Sans doute le meilleur visionneur PDF des éditeurs LaTeX" (Litteral translation to English => "Without doubt, the best PDF viewer of LaTeX editors").

What do you think of it?

PS: PDF is Masculine in French, as in "Le Document PDF".

Reply:
I still think "sans doute" is stronger than "arguably". But that's up to @cvfosammmm to decide.

Ah, "visionneur" seems to be fine: https://help.gnome.org/users/evince/stable/index.html.fr
I just found "visionneuse" on e.g. https://get.adobe.com/fr/reader/otherversions/.
It's independent from the gender of "PDF" btw.

Reply:
>  It's independent from the gender of "PDF" btw.
Yes, indeed, I confused the noun "visionneur" and adjective "visionnaire" ! And I looked more into it, and it seems that both are true, but masculine is definitely the more used one in my daily life:
- Larousse uses the feminine: https://www.larousse.fr/dictionnaires/francais/visionneuse/82203
- L'internaute uses the masculine: https://www.linternaute.fr/dictionnaire/fr/definition/visionneur/ (with a great example !)

Reply:
"Sans doute": I think that's too strong :), "Peut-être" seems fine. But I don't speak french.